introduction.md

```diff
@@ -1 +1,6 @@
-Hello, world - this is zpottie./n Ruby Path
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?